cylc/flow/commands.py

```diff
@@ -419,7 +419,7 @@ async def reload_workflow(schd: 'Scheduler', reload_global: bool = False):
         # logged by the TaskPool.
         add = set(schd.config.get_task_name_list()) - old_tasks
         for task in add:
-            LOG.warning(f"Added task: '{task}'")
+            LOG.info(f"Added task: '{task}'")
```

Comment:
Tasks which have been added by reload are currently logged at the warning level.

This shouldn't be a warning:
* This is design behaviour, Cylc has done what the user asked.
* We don't expect the user to review this list.
* There are no required actions.

We agreed to remove the opposing `Removed task` message in Tim's PR: https://github.com/cylc/cylc-flow/pull/6765/files#r2104444586